### PR TITLE
Add reduced ubuntu docker image

### PR DESCRIPTION
**Pull request by aledbf**
```
gcr.io/google_containers/ubuntu-slim 0.1    d49336ef257d        3 hours ago    55.09 MB
ubuntu                               15.10  8fba16db7a1f        13 days ago    135.5 MB
```

### CODE REVIEW

File: images/ubuntu-slim/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+Debian small docker image
```
Comment (bprashanth) [inline images/ubuntu-slim/README.md]: can you clarify *why* this is smaller?

**Comment by bprashanth** [comment]
@aledbf thanks just the one nit

**Comment by aledbf** [comment]
ping @bprashanth

**Comment by bprashanth** [comment]
Ah, cool thanks. LGTM

**Pull request merged**